repos/system_upgrade/el7toel8/actors/checksendmail/libraries/library.py

```diff
@@ -0,0 +1,25 @@
+import os
+import re
+import glob
+
+SendmailConfDir = '/etc/mail'
+SendmailConfFiles = ['/etc/mail/sendmail.cf', '/etc/mail/sendmail.mc', '/etc/mail/submit.cf', '/etc/mail/submit.mc']
+# false positives blacklist
+rfp = re.compile(r'(^\s*RIPv6:::1\b)|(@\s+\[IPv6:::1\]\s+>)')
+
+def check_false_positives(f, l):
+    return f in ["/etc/mail/sendmail.cf", "/etc/mail/submit.cf"] and rfp.search(l) is not None
+
+def check_files_for_compressed_ipv6():
+    migrate_files = []
+    g = glob.glob(SendmailConfDir + '/*.db')
+    files = list(map(lambda x: re.sub(r'\.db$', '', x), g)) + SendmailConfFiles
```

Comment:
Functional style is cool, but the agreement is not to use it unless you have really good reasons [1]
[1] https://github.com/oamg/leapp-guidelines/blob/master/python-coding-guidelines.md#7-avoid-map-reduce-and-too-complex-comprehensions
If I got the idea of L15-L16 right, collecting all *.db files from SendmailConfDir (/etc/mail) directory and appending them to the list of already known SendmailConfFiles can be done with
```
files = [os.path.join(SendmailConfDir, f.replace('.db', '')) for f in os.listdir(SendmailConfDir) if f.endswith('.db')] + SendmailConfFiles
```
Sorry if I missed something.

Reply:
I am not sure whether your code is more readable, but it's not correct, e.g. 'file.dba.db' will be recorded as '/etc/mail/filea' instead of 'file.dba'. But no problem, I will rewrite it to non-functional style.

Reply:
Fixed